packages/e2e-tests/src/specs/editor/media/insertWebMVideo.js

```diff
@@ -70,23 +74,19 @@ describe('Inserting WebM Video', () => {
 
     const fileName = await uploadMedia('small-video.webm');
     uploadedFiles.push(fileName);
-
-    await page.waitForSelector(
-      `[data-testid="mediaElement-video"] [src*="${fileName}"`
```

Comment:
How did this selector ever work? There's a missing square bracket at the end.

Reply:
This will not work at all and should be removed. This is looking for a media element is sidebar with child element with src of filename. But now we use poster images instead of video in the media library, this selector will never return anything.

Reply:
IIRC `fileName` is the file name without extension. So `[src*="${fileName}]` matches also the poster image, so it should work in theory.